docs/basics/integrations/webhooks.md

```diff
@@ -0,0 +1,80 @@
+
+
+
+## Event Payload
+
+Each event sends a POST request containing the job object with the following data fields as its payload:
+
+|Data Field|Format|Description|
+|---|---|---|
+|`id` | STRING | Sauce Labs unique identifier of the test.|
+|`creation_time` | DATE-TIME | The date-time value, in `YYYY-MM-DDTHH:mm:ss.fffZ` format, at which the test launched.|
+|`modification_time` | DATE-TIME | The date-time value, in `YYYY-MM-DDTHH:mm:ss.fffZ` format, at which the test changed. This is typically due to a change in status, which triggers a new event and new webhook post.|
+|`owner` | STRING | The Sauce Labs user who initiated the test.|
+|`owner_id` | STRING | The unique identifier of the test owner.|
+|`org_id` | STRING | The unique identifier of the Sauce Labs organization to which the test owner belongs.|
+|`team_id` | STRING | The unique identifier of the Sauce Labs team of which the test owner is a member.|
+|`team_name` | STRING | The name of the team of which the test owner is a member.|
+|`group_id` | STRING | The unique identifier of the Sauce Labs group of which the test owner is a member.|
+|`status` | ENUM | The status of the test at the time the webhook event was triggered. Possible values are:<br/><ul><li>`in-progress`</li><li>`complete`</li></ul>|
+|`passed` | BOOLEAN | If the test includes pass/fail assertions, this field states whether the test passed. Possible values are:<br/><ul><li>`1` true</li><li>`0` false</li></ul>|
+|`name` | STRING | The name of the test.|
+|`browser_name` | STRING | The browser in which the test ran.|
+|`browser_version` | INTEGER | The version of the browser in which the test ran.|
+|`os_name` | STRING | The operating system on which the test ran.|
+|`os_version` | STRING | The version of the operating system on which the test ran.|
+|`duration_sec`| INTEGER | The length of time that the test took to complete. This value is only populated in the final event push for the test.|
+|`visibility`| ENUM | Who within the Sauce Labs organization can see the test. Possible values are:<br/><ul><li>`PUBLIC`</li><li>`PUBLIC RESTRICTED`</li><li>`SHARE`</li><li>`TEAM`</li><li>`PRIVATE`</li></ul><br/>See [Sharing Test Results](/test-results/sharing-test-results/) for visibility definitions.|
+|`tags`| ARRAY | A set of string values representing custom labels for the test.|
+|`device` | STRING | For mobile app tests, the Sauce Labs unique identifier of the device on which the test was run. |
+|`app` | STRING | The App Storage identifier of uploaded test materials, such as the mobile app under test or a project test file.|
+|`error`| STRING | An error that occurred during the test.|
+|`exception`| STRING | An exception that was thrown during the test.|
+|`commit_id` | STRING | _Reserved for future integration with application versioning._ |
+|`branch_name` | STRING | _Reserved for future integration with application versioning._ |
```

Comment:
commit_id and branch_name may be populated right now and users may take advantage of it already.

Users need to pass these data in capabilities in the custom-data field.
```
"custom-data":{
         "commit":"0k392a9dkjr",
         "branch":"master",
}
``

Reply:
Oh! When I asked Lukasz about it a few weeks ago, he said it was reserved for future development. So it would look like this (java, for example):
```
DesiredCapabilities caps = DesiredCapabilities();
caps.setCapability("custom-data": { "commit":"0k392a9dkjr", "branch":"master",});
```
I am just curious for my own understanding here. Someone would potentially use this if they had their tests integrated into their CI pipeline and wanted to make sure they could distinguish each update to the code by referencing the commits? Thanks, @maciejurbanskisaucelabs

Reply:
Yes. 

@lroslonek probably said that it is for future, because we added it for Sumologic (it will be requierd to work in the future with their SDO product).